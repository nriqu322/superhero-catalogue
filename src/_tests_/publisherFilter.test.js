import React from 'react';
import { render, cleanup } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import PublisherFilter from '../components/publisherFilter';

afterEach(cleanup);

it('publisher renders ok', () => {
  const { asFragment } = render(<PublisherFilter publisherList={['Marvel Comics', 'DC Comics']} filterPublisher={() => {}} />);
  expect(asFragment()).toMatchSnapshot();
});

// it();
