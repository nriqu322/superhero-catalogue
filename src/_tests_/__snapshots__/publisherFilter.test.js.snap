// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`publisher renders ok 1`] = `
<select
  class="publisher-filter"
  name="publisher"
>
  <option>
    All
  </option>
  <option>
    Marvel Comics
  </option>
  <option>
    DC Comics
  </option>
</select>
`;

exports[`publisher representation ok 1`] = `
<select
  className="publisher-filter"
  name="publisher"
  onChange={[Function]}
>
  <option
    key="All"
  >
    All
  </option>
  <option
    key="M"
  >
    Marvel Comics
  </option>
  <option
    key="C"
  >
    DC Comics
  </option>
</select>
`;
